src/index.ts

```diff
@@ -610,17 +648,37 @@ export class Bigtable {
    * });
    *
    * //-
+   * // To control how many API requests are made and page through the results
+   * // manually, set `autoPaginate` to `false`.
+   * //-
```

Comment:
This is nothing you did, but this comment style was used for a really old version of our docs. Since we're touching it we might as well update it to use `@example <caption>...</caption>`